src/PaduaTransform.jl

```diff
@@ -0,0 +1,273 @@
+"""
+Pre-plan an Inverse Padua Transform.
+"""
+immutable IPaduaTransformPlan{DCTPLAN}
+    cfsmat::Matrix{Float64}
+    tensorvals::Matrix{Float64}
+    padvals::Vector{Float64}
+    dctplan::DCTPLAN
+end
+
+function plan_ipaduatransform(v::AbstractVector)
+    N=length(v)
+    n=Int(cld(-3+sqrt(1+8N),2))
+    @assert N==div((n+1)*(n+2),2)
+    IPaduaTransformPlan(zeros(Float64,n+2,n+1),Array(Float64,n+2,n+1),
```

Comment:
Replace `zeros` with `Array` and use `fill!`.  

Also, use `eltype(v)` instead of `Float64`.